### ISSUE TITLE
screens for mobile ui

### ISSUE THREAD

**Issue by qjflores** [qjflores/exchangemvpreact#122]

**Comment by sumukhshetty** [comment]
Responsive ?

**Comment by qjflores** [comment]
yeah responsive @joshpitzalis do you need specific designs for the screens?

**Comment by joshpitzalis** [comment]
Which screens? Just the home, login/ signup/forgot password and some static pages? I'm assuming not the rest since you said something about the tech not working on mobile. Or did I misunderstand?

**Comment by sumukhshetty** [comment]
So coinbase is coming out with this new product called token browser, which will allow web3 DApps to run on the app. 
It's around 3 days old, but it just became a part of our goals . 
Can we make the existing site responsive, with minimal changes ?

**Comment by joshpitzalis** [comment]
Well, I guess it depends how you define 'minimal'.
It's not going to be a one day fix. 

The tricky bit is that a lot of the functionality relies on tables. There is no correct way to handle tables on mobile. One way I could handle it is like this https://css-tricks.com/examples/ResponsiveTables/responsive.php (just drag the size of the window down to mobile to see what I mean).

Ideally, I would have written everything mobile first.
What I could do is, write everything mobile first from this point onwards. 
Responsiveness shouldn't prevent us from going live.
Once we have launched, I can spend a week rewriting anything that is not responsive.

Does that sound sensible or should I be pulling apart code right away so that we are responsive for launch?

Also, I'm assuming we are still sticking to chrome, or is that opening up too?

**Comment by sumukhshetty** [comment]
That sounds very sensible .
Responsive is not our priority at the moment, and it's still only chrome.
Don't worry too much about this. This would take some time to develop. But if you can write code responsive first, that would be great.
I have a feeling my designs won't be too great on a responsive site so we may need a mobile UI for that.

**Comment by sumukhshetty** [comment]
Qj, we could go with a chatbot interface for mobile as Arseniy sugggested.
Using the exisiting smart contracts to enforce a similar flow in the way it works.
